### ISSUE TITLE
Extra Shop commands?

### ISSUE THREAD

**Issue by Hoempah** [tsebring/ARK-Server-Beyond-API#2]
Hi, i also use this from you. :)

I don't know how to get this working to try it myself... :(

But i would like to have extra options for the shop commands.
ex. buy colors for dino regions with the points the shop gives you.

I think it would be a nice.

**Comment by tsebring** [comment]
## Original ARK Beyond API by Michidu

I am not the original author of ARK Beyond API. The original and the shop plugin you mentioned is made by Michidu: 

* https://github.com/Michidu/ARK-Server-Beyond-API
* https://github.com/Michidu/Ark-Server-Plugins
* http://www.beyondcheats.com/files/file/27-ark-server-beyond-api-alpha/
* http://www.beyondcheats.com/files/file/41-shop-currency-kits/

## My ARK Beyond API fork

My version of ARK Beyond API contains extra functionality required for the plugins that I have created which I have linked below.

* https://github.com/tsebring/ImprovedCommands
* https://github.com/tsebring/ArkModifiedSpawnLevelDistribution
* https://github.com/tsebring/ArkGameTestingCommands
* https://github.com/tsebring/ImprintingMod
* https://github.com/tsebring/ArkCrossServerChat

## Installation

Installation is pretty straightforward and detailed in the readme (and also on Michidu site for the original).
https://github.com/tsebring/ARK-Server-Beyond-API#installation

Basically, you download ARK Beyond API and extract it to `ARK\ShooterGame\Binaries\Win64`. If you want to use my plugins you download my version, which is also compatible with Michidu's plugins.

Then you download the plugins you want and extract them to ` ARK\ShooterGame\Binaries\Win64\BeyondApi\Plugins`. Some plugins have a configuration file `config.json` where you change settings.

If you have any specific questions or problems I will do my best to answer them here or on Discord (my tag is Tobias#5051).

**Comment by Hoempah** [comment]
Thx. I already have it running on my servers.
Someone asked if it would be possible to color dinos with the points you get from the shop.
On his site i do not see any activity anymore that's why i added your plugins which are very nice. 

I tried to open it in visual studio but it failed to build out of the box.

**Comment by tsebring** [comment]
It probably fails to build because you are missing the required include and library files, or building the wrong configuration.

Change build configuration to Release/x64.
Change include directories path for ArkApi.
Change library directories path for ArkApi.
![image](https://user-images.githubusercontent.com/408350/33472052-b062e7a2-d670-11e7-9d94-8ac9c98265e3.png)

The path where you put includes/libs should have this directory structure. All these files are found in the release archive under `/Development` except for minhook which you have to download from https://www.codeproject.com/Articles/44326/MinHook-The-Minimalistic-x-x-API-Hooking-Libra.
```
include
   API
      ...
lib
   ArkApi.lib
minhook
   include
      ...
   lib
      ...
```

**Comment by Hoempah** [comment]
Thanks.
